api/pkg/s3/abortmultipartupload.go

```diff
@@ -1,22 +1,22 @@
 package s3
 
 import (
-	"context"
-	"net/http"
+	//"context"
```

Comment:
remove these imports if not required

Reply:
ok, thanks for your suggestion, i will remove that.